src/Cli/dotnet/commands/dotnet-nuget/NuGetCommandParser.cs

```diff
@@ -63,7 +63,7 @@ private static Command GetLocalsCommand()
             var localsCommand = new Command("locals");
 
             localsCommand.AddArgument(new Argument<string>()
-                .FromAmong(new string[] { "all", "http-cache", "global-packages", "plugins-cache", "temp" }));
+                .FromAmong(new string[] { "all", "http-cache", "global-packages", "temp" }));
```

Comment:
I invoked the raw nuget CLI via `dotnet <path to nuget CLI dll> --help` and found that the list of known values for this argument in the SDK help didn't match the help from nuget directly.